app/models/trim_type.rb

```diff
@@ -0,0 +1 @@
+class TrimType < ActiveRecord::Base;end
```

Comment:
Is that actually what we have in our styleguide?  Kudos for following that.  @more-ron  does this still agree with our rubocop rules?